waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/MetaStoreProxyServer.java

```diff
@@ -187,6 +195,28 @@ private void startWaggleDance(
     LOG.info("Waggle Dance has stopped");
   }
 
+  private TServerSocket createServerSocket(boolean useSSL, int port) throws IOException, TTransportException {
```

Comment:
Why not remove useSSL as a param (we generally try avoid boolean params if possible) and retrieve the value from the conf here? You're already doing it for some other stuff in this method and you're not using the useSSL variable outside of it.

Reply:
done